### PR TITLE
add physiofit v3

### PR DESCRIPTION
**Pull request by llegregam**
Add the official PhysioFit software

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/physiofit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+about:
+  home: None
+  summary: Calculate extracellular fluxes from metabolite concentrations and biomass data
+  license: GPL-3.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - AddYourGitHubIdHere
```
Comment (bgruening) [inline recipes/physiofit/meta.yaml:51]: please add your github user here or remove this section.
Please also note that you can include a DOI here or a bio.tools identifier.

File: recipes/physiofit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  commands:
+    - pip check
+    - physiofit --help
+  requires:
+    - pip
+
+about:
+  home: None
+  summary: Calculate extracellular fluxes from metabolite concentrations and biomass data
+  license: GPL-3.0
```
Comment (bgruening) [inline recipes/physiofit/meta.yaml:46]: Please use SPDX identifiers, like this one: https://spdx.org/licenses/GPL-3.0-only.html or https://spdx.org/licenses/GPL-3.0-or-later.html

File: recipes/physiofit/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - poetry-core
+    - pip
+  run:
+    - python >3.9.7
```
Comment (bgruening) [inline recipes/physiofit/meta.yaml:25]: should that not be `>=3.8` as well?

**Review by bgruening — COMMENTED** [review]
Thanks @llegregam I have a few comments inline.

File: recipes/physiofit/meta.yaml (hunk 3 above)
Comment (llegregam) [reply recipes/physiofit/meta.yaml:25]: No actually a dependency has problems with python 3.9.7 so I decided to get a greater than version

**Review by llegregam — COMMENTED** [review]
REquested changes have been made

**Review by mencian — CHANGES_REQUESTED** [review]

File: recipes/physiofit/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  imports:
+    - physiofit
+  commands:
+    - pip check
+    - physiofit --help
+  requires:
+    - pip
+
+about:
+  home: None
```
Comment (mencian) [inline recipes/physiofit/meta.yaml:44]:
```suggestion
  home: https://github.com/MetaSys-LISBP/PhysioFit
```

File: recipes/physiofit/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  summary: Calculate extracellular fluxes from metabolite concentrations and biomass data
+  license: GPL-3.0-only
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - github.com/llegregam
```
Comment (mencian) [inline recipes/physiofit/meta.yaml:51]:
```suggestion
    - llegregam
```

**Review by mencian — CHANGES_REQUESTED** [review]
Thanks, a couple of comments inline that I previously overlooked. Otherwise, looks good to me!

**Review by mencian — APPROVED** [review]

**Pull request merged**